js/ui/form/ui.form.js

```diff
@@ -991,6 +991,10 @@ const Form = Widget.inherit({
                     }
                 }
                 if(optionName === 'visible') { // T874843
+                    if(layoutManager.option(fullOptionName) === value) {
```

Comment:
no
see https://hg/mobile/rev/85c90979eb7c

Reply:
Yep, you are absolutely right.
Prepared a new variant of fix - more simple